jsf/subsystem/src/test/resources/modules/org/jboss/as/jsf-injection/1.2/module.xml

```diff
@@ -33,9 +33,10 @@
     </resources>
```

Comment:
We could probably remove this since JSF 1.2 won't work anymore here.

Reply:
Done